server/fleet/activities.go

```diff
@@ -1317,6 +1317,56 @@ func (a ActivityTypeWipedHost) Documentation() (activity, details, detailsExampl
 }`
 }
 
+type ActivityTypeCreatedDeclarationProfile struct {
```

Comment:
Oh just one thing I realized after approving, you need to generate the activities docs.

Reply:
thanks for the reminder! I'm considering sending those in a completely separate PR (even from the feature branch PR) given the hurry to merge :(